app/controllers/concerns/authentication_and_sso_concerns.rb

```diff
@@ -45,15 +46,22 @@ def load_user
     @current_user = User.find(@session_object.uuid) if @session_object
   end
 
-  # Destroys the users session in 1) Redis, 1) the MHV SSO Cookie, 3) and the Session Cookie
-  def reset_session
-    Rails.logger.info('SSO: ApplicationController#reset_session', sso_logging_info)
+  # Destroys the users session in 1) Redis and the MHV SSO Cookie
+  def clear_session
+    Rails.logger.info('SSO: ApplicationController#clear_session', sso_logging_info)
```

Comment:
Same concern as above about this being "noisy".  For `reset_session` the `sso_logging_info` still gets logged (and in fact gets logged twice as it currently is since `reset_session` calls `clear_session`)

Reply:
Shall I just delete that logging statement?